.github/workflows/end2end.yaml

```diff
@@ -66,6 +67,7 @@ env:
   AWS_BUCKET_NAME: ci-zenko-aws-target-bucket
   AWS_CRR_BUCKET_NAME: ci-zenko-aws-crr-target-bucket
   AWS_FAIL_BUCKET_NAME: ci-zenko-aws-fail-target-bucket
+  AWS_REPLICATION_CTST_BUCKET_NAME: ci-zenko-aws-replication-ctst-bucket
```

Comment:
would be great if we can just use AWS_BACKEND_DESTINATION_LOCATION and AWS_CRR_BUCKET_NAME